test/invoice-workflow.e2e-spec.ts

```
@@ -0,0 +1,884 @@
+
+      const customerId = 'customer-status-api-test';
+
+      console.log('🔧 Creating workflow through public API...');
+
+      const workflow1Data = TestDataGenerator.generateStartWorkflowData();
+      workflow1Data.customerId = customerId;
+
+      const workflow2Data = TestDataGenerator.generateStartWorkflowData();
+      workflow2Data.customerId = customerId;
+
+      const response1 = await request(app.getHttpServer())
```

ESLint

[error] 'response1' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)